fdbclient/MutationLogReader.actor.cpp

```diff
@@ -103,7 +103,7 @@ ACTOR Future<Void> PipelinedReader::getNext_impl(PipelinedReader* self, Database
 					return Void();
 				}
 
-				begin = kvs.readThrough.present() ? kvs.readThrough.get() : keyAfter(kvs.back().key);
+				begin = kvs.getReadThrough(begin.arena());
```

Comment:
This is going to add a copy when we aren't using `readThrough`, since it is going to write the value into `begin.arena()` but then assign from a `KeyRef` -> `Key`.

Reply:
Great point, updated with non arena parameter and return `Key` directly.